main.yml

```
@@ -845,7 +845,11 @@ parameters:
                   on(organization)
                   group_left(sales_order)
                   (
-                    appuio_control_organization_info{namespace="appuio-control-api-production"}
+                    # Drop pod and instance labels so we gracefully handle
+                    # short time periods (e.g. during maintenance) where it's
+                    # possible that metrics are scraped from two control-api
+                    # pods.
+                    max without (pod,instance) appuio_control_organization_info{namespace="appuio-control-api-production"}
```

```suggestion
                    max without(pod,instance) appuio_control_organization_info{namespace="appuio-control-api-production"}
```

Might need to be written like this. Can't be parsed by promtool for the tests.

---

Actually, I messed up the syntax, the metric needs to be in parentheses

```suggestion
                    max without (pod,instance) (appuio_control_organization_info{namespace="appuio-control-api-production"})
```